packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -757,7 +759,7 @@ An invalid SCIM filter returns the HTTP 400 error with a description of the issu
 ```json
 {
   "errorCode": "E0000053",
-  "errorSummary": "Invalid filter 'display_message eqq \"Create okta user\"': Unrecognized attribute operator 'eqq' at position 16. Expected: eq,co,sw,pr,gt,ge,lt,le",
+  "errorSummary": "Invalid filter 'display_message eq \"Create okta user\"': Unrecognized attribute operator 'eqq' at position 16. Expected: eq,co,sw,pr,gt,ge,lt,le",
```

Comment:
Are you sure you want to replace `eqq` with `eq`? Looks like that is the error here.

Reply:
`eqq` is not a valid operator.

Reply:
Oh totally missed that, you're right! Will revert.